src/test/HostActivationTests/FrameworkResolution/RollForwardOnNoCandidateFxSettings.cs

```diff
@@ -198,6 +198,74 @@ public void NoInheritance_MoreRestrictive(SettingLocation settingLocation, strin
                 resolvedFramework);
         }
 
+        [Theory]
+        [InlineData("1.0.0", "2.5.5", "2.5.5")]
+        [InlineData("2.0.0", "2.5.5", "2.5.5")]
+        [InlineData("2.5.4", "2.5.5", "2.5.5")]
+        [InlineData("2.5.5", "2.5.5", "2.5.5")]
+        [InlineData("2.5.5", "2.5.4", "2.5.4")]
+        [InlineData("2.5.5", "2.5.3", null)]
+        [InlineData("2.5.5", "2.5.5-preview1", null)]
+        public void FxVersionCLIOverride(string frameworkReferenceVersion, string fxVersion, string resolvedFramework)
+        {
+            RunTestWithRollForwardOnNoCandidateFxSetting(
+                new TestSettings()
+                    .WithRuntimeConfigCustomizer(runtimeConfig => runtimeConfig
+                        .WithFramework(MicrosoftNETCoreApp, frameworkReferenceVersion))
+                    .WithCommandLine(Constants.FxVersion.CommandLineArgument, fxVersion),
+                resolvedFramework: resolvedFramework);
+        }
+
+        [Theory]
+        [InlineData(null, null)]
+        [InlineData(0, null)]
+        [InlineData(0, true)]
+        [InlineData(1, null)]
+        [InlineData(1, true)]
+        [InlineData(2, null)]
+        [InlineData(0, false)]
+        public void FxVersionCLIIgnoresOtherSettings(int? rollForwardOnNoCandidateFx, bool? applyPatches)
```

Comment:
Maybe make `FxVersionCLI_` the prefix? e.g. `FxVersionCLI_IgnoresOtherSettings`, `FxVersionCLI_AppliesToFirstFrameworkReference_NETCoreAppFirst`
Seems more in line with the naming of the other framework resolution tests.

Reply:
I moved them into a separate class, so no prefix at all.